addons/sale_pdf_quote_builder/models/ir_actions_report.py

```diff
@@ -43,56 +52,77 @@ def _render_qweb_pdf_prepare_streams(self, report_ref, data, res_ids=None):
                 if (not has_header and not included_product_docs and not has_footer):
                     continue
 
+                all_form_fields = set()
                 writer = PdfFileWriter()
+
                 if has_header:
-                    self._add_pages_to_writer(writer, base64.b64decode(header_record.sale_header))
+                    decoded_header = base64.b64decode(header_record.sale_header)
+                    self._add_pages_to_writer(writer, decoded_header, all_form_fields)
                 if included_product_docs:
                     for doc in included_product_docs:
+                        decoded_doc = base64.b64decode(doc.datas)
+                        sol_id = doc_line_id_mapping[doc.id]
                         self._add_pages_to_writer(
-                            writer, base64.b64decode(doc.datas), doc_line_id_mapping[doc.id]
+                            writer, decoded_doc, all_form_fields, sol_id=sol_id
                         )
                 self._add_pages_to_writer(writer, initial_stream.getvalue())
                 if has_footer:
-                    self._add_pages_to_writer(writer, base64.b64decode(footer_record.sale_footer))
+                    decoded_footer = base64.b64decode(header_record.sale_footer)
```

Comment:
```suggestion
                    decoded_footer = base64.b64decode(footer_record.sale_footer)
```
To not forget it, will change it during the review